pkg/core/interop_neo.go

```diff
@@ -19,6 +19,8 @@ const (
 	FindRemovePrefix = 1 << 1
 	FindValuesOnly   = 1 << 2
 	FindDeserialize  = 1 << 3
+	FindPick0        = 1 << 4
+	FindPick1        = 1 << 5
```

Comment:
Don't we also need to declare these flags as FindFlags (at `pkg/interop/storage/storage.go`)?

Reply:
Fixed.